domains/fullsaglamindirsene.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "ScorBoss",
+           "email": "alibebek09@gmail.com",
+           "discord": "702241885967876216"
+        },
+    
+        "record": {
+            "CNAME": "www.ghs.google.com"
```

Comment:
Since I assume this is for Google BlogSpot (smth like that) please give a preview of your website. Also don't you also need to make another domain for this?